homework-projects/CharlesDube/Index index.adoc

```diff
@@ -0,0 +1,14 @@
+[[index-file-template]]
+= Index file template
+:toc: left
+:toclevels: 3
+
+*Replace me:* This index file template contains two included files listed below.
+
+include::Draft Part 1.adoc[leveloffset=+1]
+
+include::Draft Part 2.adoc[leveloffset=+1]
+
+include::https://media.tenor.com/OqvDDg1mwCMAAAAC/i-dont-know-what-the-hell-im-doing-matt-logelin.gif[leveloffset=+1]
```

Comment:
incorrect syntax for including a link. See more details: https://docs.asciidoctor.org/asciidoc/latest/macros/links/